ssl/ssl_lib.cc

```diff
@@ -929,6 +931,24 @@ static int ssl_do_post_handshake(SSL *ssl, const SSLMessage &msg) {
   return 1;
 }
 
+// Sets pha_enabled value in SSL_CTX, 0 is disabled, 1 is enabled
+void SSL_CTX_set_post_handshake_auth(SSL_CTX *ctx, int val) {
+  ctx->pha_enabled = val;
+}
+
+// Sets pha_enabled value in SSL, 0 is disabled, 1 is enabled
```

Comment:
Minor Nit: Comments here are a bit redundant, since standard C++/C indicate 1 for true and 0 for false. :)